lib/auth/accountrecovery.go

```diff
@@ -102,11 +88,10 @@ func (a *Server) StartAccountRecovery(ctx context.Context, req *proto.StartAccou
 	return token, nil
 }
 
-// verifyCodeWithRecoveryLock counts number of failed attempts at providing a valid recovery code.
-// After MaxAccountRecoveryAttempts, user is temporarily locked from further attempts at recovering and also
-// locked from logging in. Modeled after existing function WithUserLock.
+// verifyCodeWithRecoveryLock validates the recovery code for the user and will unlock their account if
+// the code is valid.  If the code is invalid, a failed recovery attempt will be recorded.
 func (a *Server) verifyCodeWithRecoveryLock(ctx context.Context, username string, recoveryCode []byte) error {
-	user, err := a.Services.GetUser(ctx, username, false)
+	_, err := a.Services.GetUser(ctx, username, false)
```

Comment:
the method name `verifyCodeWithRecoveryLock` is no longer correct since we don't lock anymore. maybe... `verifyCodeWithRecord`?

Reply:
Updated, thank you!